Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -125,15 +126,29 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
 
             // Skip properties that have both getters and setters to not interfere
             // with other post-processors in the chain that initialize them later.
-            var getMethods = type.GetProperties()
-                                    .Where(p => p.GetGetMethod() != null &&
-                                                p.GetSetMethod() != null)
-                                    .Select(p => p.GetGetMethod());
-            methods = methods.Except(getMethods);
+            methods = RemoveGetterSetterMethods(type, methods);
 
             return methods.Where(CanBeConfigured);
         }
 
+        private static IEnumerable<MethodInfo> RemoveGetterSetterMethods(Type type, IEnumerable<MethodInfo> methods)
```

Comment:
Probably, better name `SkipWritablePropertyGetters`:
- in fact, you not remove something, but filter that
- you don't filter out setters

Reply:
OK